storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -407,6 +407,23 @@ static bool check_server_version(unsigned long version_number,
     }
   }
 
+  std::string mysql_version = MYSQL_SERVER_VERSION;
+
+  // replace 8.0.21 first . with 0 and second . with null
+  std::size_t found = mysql_version.find_first_of(".");
+  if (found != std::string::npos) mysql_version[found] = '0';
```

Comment:
This assumes PXB will always have x.x.xx version pattern, which will require to re-work this function in future. For example, I did a test with PS 8.10.22 and PXB 8.10.21:
```
(gdb) p mysql_version_int
+p mysql_version_int
$3 = 801021
(gdb) p version_number
+p version_number
$4 = 81022
```
I would make this function to use the same logic as mysql_get_server_version, which is where `version_number` is extracted from.